lib/gollum/views/template_cascade.rb

```diff
@@ -0,0 +1,35 @@
+module Precious
+  module Views
+    module TemplateCascade
+      def template_priority_path
+        @@template_priority_path
+      end
+
+      def template_priority_path=(path)
+        @@template_priority_path = File.expand_path(path)
+        @template = nil
+      end
+
+      def first_path_available(name)
+        priority = "#{template_priority_path}/#{name}.#{template_extension}"
```

Comment:
Please use `::File.join()` here instead of the hardcoded slashes. I think in practice slashes work on Windows, but better to make sure! Same on the next line.

Reply:
No problem. This is how the parent Mustache class does it so I had a choice of whose patterns to respect.